src/glwidget.cpp

```diff
@@ -104,7 +104,7 @@ GLWidget::GLWidget(TMap* pM, QWidget* parent)
 , mScale()
 , mTarget()
 {
-    mpHost = 0;
+    mpHost = nullptr;
```

Comment:
If I recall correctly only one of these constructors is actually used, and the initialization of `mpHost` is better done in the initialiser list IMHO...